src/PeopleTable.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { PersonRow } from './PersonRow';
+
+export const PeopleTable = ({ peoples }) => (
+  <table className="table">
+    <tr>
```

Comment:
Use tags `thead` for headings and `tbody` for all other rows. They are semantical tags for `table` element